lib/active_fedora/scoping/named.rb

```diff
@@ -13,6 +13,7 @@ module ClassMethods
         #
         #   fruits = Fruit.all
         #   fruits = fruits.where(color: 'red') if options[:red_only]
+        #   fruits = fruits.where(size: ['big', 'little'], join_with: ' OR ') #gets big or little fruit
```

Comment:
In activerecord, passing an array, always produces an `IN` query. This is equivalent to an `OR` in solr.  I think we should do away with the `join_with` argument and say, if you pass in an array, you want an or.  Here's how AR works:

```ruby
User.where(email: ['foo', 'bar'])
  SELECT "users".* FROM "users" WHERE "users"."email" IN ('foo', 'bar')
```